scripts/dockerfiles/rsyslog_conf/etc/logrotate.d/rsyslog

```diff
@@ -1,14 +1,13 @@
 /var/log/syslog
 {
 	rotate 7
-	size 5M
+	maxsize=10M
+	minsize=9M
```

Comment:
Just consistency.
I am actually skeptic about the `==`
I don't mind being explicit about it